src/main/groovy/org/mockito/release/internal/gradle/InitConfigFileTask.java

```diff
@@ -0,0 +1,75 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.TaskAction;
+import org.mockito.release.notes.util.IOUtil;
+
+import java.io.File;
+
+public class InitConfigFileTask extends DefaultTask{
+
+    private static final Logger LOG = Logging.getLogger(InitConfigFileTask.class);
+
+    private File configFile;
+
+    @TaskAction public void initShipkitConfigFile(){
+        if(configFile.exists()){
+            LOG.lifecycle("  File {} already exists. Nothing to create.", configFile.getPath());
+        } else{
+            createShipKitConfigFile();
+            LOG.lifecycle("  Created config file at {}. Please configure it.", configFile.getPath());
+        }
+    }
+
+    private void createShipKitConfigFile() {
+        String content =
+                new TemplateResolver(DEFAULT_SHIPKIT_CONFIG_FILE_CONTENT)
+                        .withProperty("gitHub.repository", "mockito/mockito-release-tools-example")
+                        .withProperty("gitHub.writeAuthUser", "shipkit")
```

Comment:
I was surprised that this had worked. GitHub auth token seems to ignore the name of the user even though you need to specify one when doing git push. Basically you can do ```git push foobar:<auth token>@somewhere master``` and 'foobar' will be happily accepted so long the auth token is right.

Let's see how it goes and if GitHub does not change it.